web/source/text/defaultOutput.ts

```diff
@@ -25,15 +25,13 @@ namespace com.keyman.text {
      * Serves as a default keycode lookup table.  This may be referenced safely by mnemonic handling without fear of side-effects.
      * Also used by Processor.defaultRuleBehavior to generate output after filtering for special cases.
      */
-    public static forAny(Lkc: KeyEvent) {
+    public static forAny(Lkc: KeyEvent, isMnemonic: boolean) {
       var char = '';
 
-      let code = DefaultOutput.codeForEvent(Lkc);
-
       // A pretty simple table of lookups, corresponding VERY closely to the original defaultKeyOutput.
       if(char = DefaultOutput.forSpecialEmulation(Lkc)) {
```

Comment:
One thing I didn't note last time is that this pattern is a red flag for linters (or casual code reviewers) -- single equals sign in an if statement looks like a mistake (I know it isn't). But it might be good to qualify it, even though it's technically unnecessary.

```suggestion
      if((char = DefaultOutput.forSpecialEmulation(Lkc)) !== '') {
```

Reply:
Yeah, that makes sense.

It did feel a little odd to use, though it was clearly the most concise way to do things.  (The if-ladder was a lot uglier otherwise.)  This is a bit more verbose, but just a bit, and it keeps things organized just as cleanly.